dbms/src/Storages/Page/V3/Universal/UniversalWriteBatchImpl.h

```diff
@@ -103,20 +106,56 @@ class UniversalWriteBatch : private boost::noncopyable
     void putExternal(const UniversalPageId & page_id, UInt64 tag)
     {
         // External page's data is not managed by PageStorage, which means data is empty.
-        Write w{WriteBatchWriteType::PUT_EXTERNAL, page_id, tag, nullptr, 0, "", {}};
+        Write w{WriteBatchWriteType::PUT_EXTERNAL, page_id, tag, nullptr, 0, "", {}, std::nullopt};
         writes.emplace_back(std::move(w));
     }
 
+    void putRemotePage(PageIdU64 page_id, const PS::V3::CheckpointLocation & loc, PageSize size, const PageFieldSizes & data_sizes = {})
```

Comment:
I can't find the place this method is used.

Reply:
removed